server/routes/about-person/routes.ts

```diff
@@ -1,9 +1,9 @@
 import { Router } from 'express'
-import AboutPopController from './AboutPopController'
+import AboutPersonController from './AboutPersonController'
 import URLs from '../URLs'
 
 export default function setupAboutPopRoutes(router: Router) {
-  const controller = new AboutPopController()
+  const controller = new AboutPersonController()
 
   router.get(URLs.ABOUT_POP, controller.get)
```

Comment:
`_POP`

Reply:
My main goal was getting the file structure consistent. I'll rename ABOUT_POP though and have a look through the other `pop`s.